error occurred in cc-rs: unknown target `x86_64-chimera-linux-musl`.
NOTE: `cc-rs` only supports a fixed set of targets when not in a build script.
- If adding a new target, you will need to fork of `cc-rs` until the target
has landed on nightly and the auto-generated list has been updated. See also
the `rustc` dev guide on adding a new target:
https://rustc-dev-guide.rust-lang.org/building/new-target.html
- If using a custom target, prefer to upstream it to `rustc` if possible,
otherwise open an issue with `cc-rs`:
https://github.com/rust-lang/cc-rs/issues/new

diff --git a/vendor/cc/src/target/generated.rs b/vendor/cc/src/target/generated.rs
index bfaa2937..fdff3e4e 100644
--- a/vendor/cc/src/target/generated.rs
+++ b/vendor/cc/src/target/generated.rs
@@ -124,6 +124,18 @@ pub(crate) const LIST: &[(&str, TargetInfo<'static>)] = &[
             unversioned_llvm_target: "arm64-apple-watchos-simulator",
         },
     ),
+    (
+        "aarch64-chimera-linux-musl",
+        TargetInfo {
+            full_arch: "aarch64",
+            arch: "aarch64",
+            vendor: "chimera",
+            os: "linux",
+            env: "musl",
+            abi: "",
+            unversioned_llvm_target: "aarch64-chimera-linux-musl",
+        },
+    ),
     (
         "aarch64-fuchsia",
         TargetInfo {
@@ -748,6 +760,18 @@ pub(crate) const LIST: &[(&str, TargetInfo<'static>)] = &[
             unversioned_llvm_target: "armv7-apple-ios",
         },
     ),
+    (
+        "armv7-chimera-linux-musleabihf",
+        TargetInfo {
+            full_arch: "armv7",
+            arch: "arm",
+            vendor: "chimera",
+            os: "linux",
+            env: "musl",
+            abi: "eabihf",
+            unversioned_llvm_target: "armv7-chimera-linux-musleabihf",
+        },
+    ),
     (
         "armv7-linux-androideabi",
         TargetInfo {
@@ -1420,6 +1444,18 @@ pub(crate) const LIST: &[(&str, TargetInfo<'static>)] = &[
             unversioned_llvm_target: "i686-unknown-linux-gnu",
         },
     ),
+    (
+        "loongarch64-chimera-linux-musl",
+        TargetInfo {
+            full_arch: "loongarch64",
+            arch: "loongarch64",
+            vendor: "chimera",
+            os: "linux",
+            env: "musl",
+            abi: "",
+            unversioned_llvm_target: "loongarch64-chimera-linux-musl",
+        },
+    ),
     (
         "loongarch64-unknown-linux-gnu",
         TargetInfo {
@@ -1780,6 +1816,18 @@ pub(crate) const LIST: &[(&str, TargetInfo<'static>)] = &[
             unversioned_llvm_target: "nvptx64-nvidia-cuda",
         },
     ),
+    (
+        "powerpc-chimera-linux-musl",
+        TargetInfo {
+            full_arch: "powerpc",
+            arch: "powerpc",
+            vendor: "chimera",
+            os: "linux",
+            env: "musl",
+            abi: "",
+            unversioned_llvm_target: "powerpc-chimera-linux-musl",
+        },
+    ),
     (
         "powerpc-unknown-freebsd",
         TargetInfo {
@@ -1888,6 +1936,18 @@ pub(crate) const LIST: &[(&str, TargetInfo<'static>)] = &[
             unversioned_llvm_target: "powerpc-unknown-linux-gnuspe",
         },
     ),
+    (
+        "powerpc64-chimera-linux-musl",
+        TargetInfo {
+            full_arch: "powerpc64",
+            arch: "powerpc64",
+            vendor: "chimera",
+            os: "linux",
+            env: "musl",
+            abi: "",
+            unversioned_llvm_target: "powerpc64-chimera-linux-musl",
+        },
+    ),
     (
         "powerpc64-ibm-aix",
         TargetInfo {
@@ -1960,6 +2020,18 @@ pub(crate) const LIST: &[(&str, TargetInfo<'static>)] = &[
             unversioned_llvm_target: "powerpc64-unknown-linux-gnu",
         },
     ),
+    (
+        "powerpc64le-chimera-linux-musl",
+        TargetInfo {
+            full_arch: "powerpc64le",
+            arch: "powerpc64",
+            vendor: "chimera",
+            os: "linux",
+            env: "musl",
+            abi: "",
+            unversioned_llvm_target: "powerpc64le-chimera-linux-musl",
+        },
+    ),
     (
         "powerpc64le-unknown-freebsd",
         TargetInfo {
@@ -2236,6 +2308,18 @@ pub(crate) const LIST: &[(&str, TargetInfo<'static>)] = &[
             unversioned_llvm_target: "riscv32",
         },
     ),
+    (
+        "riscv64-chimera-linux-musl",
+        TargetInfo {
+            full_arch: "riscv64",
+            arch: "riscv64",
+            vendor: "chimera",
+            os: "linux",
+            env: "musl",
+            abi: "",
+            unversioned_llvm_target: "riscv64-chimera-linux-musl",
+        },
+    ),
     (
         "riscv64-linux-android",
         TargetInfo {
@@ -2896,6 +2980,18 @@ pub(crate) const LIST: &[(&str, TargetInfo<'static>)] = &[
             unversioned_llvm_target: "x86_64-apple-watchos-simulator",
         },
     ),
+    (
+        "x86_64-chimera-linux-musl",
+        TargetInfo {
+            full_arch: "x86_64",
+            arch: "x86_64",
+            vendor: "chimera",
+            os: "linux",
+            env: "musl",
+            abi: "",
+            unversioned_llvm_target: "x86_64-chimera-linux-musl",
+        },
+    ),
     (
         "x86_64-fortanix-unknown-sgx",
         TargetInfo {
